HomeWork/src/HW9/Professions/Person.java

```diff
@@ -0,0 +1,31 @@
+package HW9.Professions;
+
+public class Person {
+    private String nameFS;
+    private int experience;
```

Comment:
experience should be described in Driver class, if we are talking about driving experience